src/events.rs

```diff
@@ -101,7 +101,11 @@ pub enum WindowEvent {
 pub enum DeviceEvent {
     Added,
     Removed,
+    /// In the case of mouse motion, the axis is 0 (in x) or 1 (in y), and the value is the distance delta.
```

Comment:
This could be phrased better. For example, "Mouse devices yield `Motion` events where axis 0 is horizontal and axis 1 is vertical." It should go without saying that `value` is the quantity of motion.